src/scripts/smoothScroll/scrollModule.js

```diff
@@ -0,0 +1,74 @@
+import easingOptions from "./easingOptions.js";
+
+const defaultOptions = {
+  duration: 2500,
+  animation: easingOptions.easeOutCubic,
+};
+
+function Scroll(opts) {
+  this.options = {
+    class: opts.class,
+    animation: opts.animation,
```

Comment:
What about replacing animation and duration initial declarations with a default assignment?
```
animation: opts.animation || defaultOptions.animation,
duration: opts.duration || defaultOptions.duration,
```

We call `_setDefaults` during `init` and at the end of the animation. We could avoid those two assignments inside that method.